src/js_types.hpp

```diff
@@ -397,6 +397,15 @@ inline bool Value<T>::is_valid_for_property(ContextType context, const ValueType
 
 inline std::string js_type_name_for_property_type(PropertyType type)
 {
+    if (realm::is_array(type)) {
+       if (type == PropertyType::LinkingObjects) {
+            throw std::runtime_error("LinkingObjects' type is not supported");
```

Comment:
Hmm, wasn't support added for linking objects in the last few months? Surely this should now be returning a valid string rather than throwing?